ui/src/screen/project/source.ts

```diff
@@ -61,6 +61,17 @@ interface Screen {
   tree: Writable<source.Tree>;
 }
 
+export interface CommitsHistory {
+  history: CommitGroup[];
+  stats: source.Stats;
+}
```

Comment:
To avoid having two different types with the same name I’d suggest to call this `GroupedCommitHistory`.

Reply:
Done!